src/constants/networks.ts

```diff
@@ -4,3 +4,17 @@ export const Futurenet = {
   networkPassphrase: 'Test SDF Future Network ; October 2022',
   sorobanRpcUrl: 'https://rpc-futurenet.stellar.org',
 };
+
+export const Mainnet = {
+  network: 'PUBLIC',
+  networkUrl: 'https://horizon.stellar.org',
+  networkPassphrase: 'Public Global Stellar Network ; September 2015',
+  sorobanRpcUrl: 'https://soroban-testnet.stellar.org', // TODO: change to actual mainnet RPC
```

Comment:
add `contract` field here and remove the src/constants/contract.ts and use contract address from here